Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml

```diff
@@ -37,3 +37,195 @@
     sprite: DeltaV/Structures/Walls/railing.rsi
   - type: Icon
     sprite: DeltaV/Structures/Walls/railing.rsi
+
+- type: entity
+  parent: Railing
+  id: RailingDeltaVGrey
+  suffix: Modern
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: side_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: side_grey
+
+- type: entity
+  parent: RailingCorner
+  id: RailingCornerDeltaVGrey
+  suffix: Modern
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_grey
+
+- type: entity
+  parent: RailingCornerSmall
+  id: RailingCornerSmallDeltaVGrey
+  suffix: Modern
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_small_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_small_grey
+
+- type: entity
+  parent: RailingRound
+  id: RailingRoundDeltaVGrey
+  suffix: Modern
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: round_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: round_grey
+
+- type: entity
+  parent: Railing
+  id: RailingDeltaVWood
+  suffix: Modern
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: side_wood
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: side_wood
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 100
+      behaviors:
+        - !type:DoActsBehavior
+          acts: [ "Destruction" ]
+    - trigger:
+        !type:DamageTrigger
+        damage: 20
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          collection: WoodDestroy
+          params:
+            volume: -6
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MaterialWoodPlank1:
+            min: 0
+            max: 1
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
```

Comment:
put this into a `BaseRailingWood` abstract prototype and add it to the wood railings parents

Reply:
oh shit right we can multi parent